projects/igniteui-angular/src/lib/tabs/tabs/tab-header.component.ts

```diff
@@ -115,9 +115,11 @@ export class IgxTabHeaderComponent extends IgxTabHeaderDirective implements Afte
 
     /** @hidden @internal */
     public ngOnDestroy(): void {
-        this.ngZone.runOutsideAngular(() => {
-            this._resizeObserver.disconnect();
-        });
+        if (this._resizeObserver) {
+            this.ngZone.runOutsideAngular(() => {
+                this._resizeObserver.disconnect();
+            });
+        }

```

Comment:
```suggestion
        this.ngZone.runOutsideAngular(() => {
            this._resizeObserver?.disconnect();
        });
```

Reply:
My idea was to completely avoid the `runOutsideAngular` call (as the `disconnect` is the only thing being done there).

What you suggested should do the trick as well, so I can change it to your version if your prefer it.

Reply:
I don't think the empty callback would affect performance or anything else, but the code looks cleaner this way.

Reply:
Good point, thanks for your feedback. I will change it to your suggestion.